rose/server/web/index.html

```diff
@@ -9,11 +9,20 @@
 
   <body>
     <div id="content">
+      <a href="settings.html" id="settings-link">Settings</a>
+      <a href="#" id="info-btn">Info</a>
+
+      <div id="info-panel" class="hidden">
```

Comment:
class=hidden vs hidden vs use a pre-built custom-element with aria announce stuff ready to go

Reply:
ARIA is a little out of scope here :-)
we may implement when we move to web components

Reply:
While aria is not necessary, the experience must be available to screen reader users